modules/core/shared/src/main/scala/gem/Observation.scala

```diff
@@ -89,15 +94,30 @@ object Observation {
 
   }
 
-  /** Observation is a bitraversable functor. */
-  implicit val ObservationBitraverse: Bitraverse[Observation] =
-    new Bitraverse[Observation] {
-      def bifoldLeft[A, B, C](fab: Observation[A,B], c: C)(f: (C, A) => C, g: (C, B) => C): C =
-        fab.steps.foldLeft(f(c, fab.staticConfig))(g)
-      def bifoldRight[A, B, C](fab: Observation[A,B], c: Eval[C])(f: (A, Eval[C]) => Eval[C], g: (B, Eval[C]) => Eval[C]): Eval[C] =
-        fab.steps.foldRight(f(fab.staticConfig, c))(g)
-      def bitraverse[G[_]: Applicative, A, B, C, D](fab: Observation[A, B])(f: A => G[C], g: B => G[D]): G[Observation[C,D]] =
-        (f(fab.staticConfig), fab.steps.traverse(g)).mapN((c, d) => fab.copy(staticConfig = c, steps = d))
+  /** A functor over `Observation` on the `T`, or targets, type parameter.
+    * Not implicit.
+    */
+  def targetsFunctor[S, D]: Functor[Observation[?, S, D]] =
```

Comment:
I think this approach is sensible, but can we strengthen these to `Traverse`? I can do a followup PR if you like.